mgmtsystem_nonconformity/readme/USAGE.rst

```diff
@@ -0,0 +1,45 @@
+To use this module:
+
+* Go to Management System > Nonconformities
+* Click on Create to enter the following information:
+
+* Partner : Customer, supplier or internal personnel
+* Related to: Any reference pointing to the NC (order id, project id, task id, etc.)
+* Responsible: Person responsible for the NC
+* Manager : Person managing the department or owner of the procedure
+* Filled in by: Originator of NC report
+* Origins:  The source of the NC, how was it discover
+* Procedures:  Against which procedure is the NC
+* Description: Evidence, reference to the standards
+
+* Click on Save and then on Analysis.
```

Comment:
For me better:
* Click on Save button and then on "Analysis" on stage sequence in the top right corner.